adserver/admin.py

```diff
@@ -14,6 +14,8 @@
 from djstripe.models import Invoice
 from simple_history.admin import SimpleHistoryAdmin
 
+from adserver.analyzer.models import AnalyzedAdvertiserUrl
```

Comment:
I do wonder about this for folks who have the analyzer disabled.

Reply:
I think we should probably just fully migrate the analyzer over to ext, tbh. There's definitely a bunch of split brain stuff happening in this PR, and I think that's likely the cleanest thing.